doc/vast.1.md

```diff
@@ -59,31 +59,60 @@ The following key components exist:
   Accepts query expressions from users, asks **index** for hits, takes them to
   **archive** to extract candidates, and relays matching events to **sink**s.
 
+### Schematic
+```
+                +--------------------------------------------+
+                | Node                                       |
+                |                                            |
+  +--------+    |             +--------+                     |    +-------+
+  | source |    |         +--->archive <------+           +-------> sink  |
+  +-----bro+-------+      |   +--------<---+  v-----------++ |    +---json+
+                |  |      |                |  | exporter   | |
+                | +v------++           +------>------------+ |
+     ...        | |importer|           |   |     ...         |      ...
+                | +^------++           |   |                 |
+                |  |      |            |   +-->------------+ |
+  +--------+-------+      |            |      | exporter   | |
+  | source |    |         |   +--------v      ^-----------++ |    +-------+
+  +----pcap+    |         +---> index  <------+           +-------> sink  |
+                |             +--------+                     |    +--ascii+
+                |                                            |
+                |                                            |
+                +--------------------------------------------+
+```
+The above diagram illustrates the default configuration of a single node and
+the flow of messages between the components. The **importer**, **index**, and
+**archive** are singleton instances within the **node**. **Source**s are spawned
+on demand for each ingestion. **Sink**s and **exporter**s form pairs that are
+spawned on demand for each database query. **Source**s and **sink**s exist in their
```

Comment:
> [..] for each database query.

I'd remove "query" here.

Reply:
Then the sentence doesn't make sense any more?

Reply:
D'oh, I meant "database" actually. 😅